doomrainbow/src/main/java/com/detroitlabs/kyleofori/doomrainbow/RainbowView.java

```diff
@@ -110,9 +133,9 @@ private static void initDefaultForegroundArcPaint() {
 
     /**
      * Aspect ratio of child view, such that
-     *
-     *     w = LAMBDA h
-     *
+     * <p/>
```

Comment:
I actually didn't add that.  It must have happened when I automatically formatted this class.